### ISSUE TITLE
Create a calendar of events

### ISSUE THREAD

**Issue by carolynbowes** [massgov/feature-requests#36]
**User story:**
We’d love to find a better way to display our upcoming events in a calendar format, and have the ability to display a lot of information within this calendar. Examples of content that this would be useful for includes:

- https://www.mass.gov/service-details/basic-hunter-education-course-offerings
- http://www.mass.gov/agr/massgrown/calendar.htm
- https://www.mass.gov/service-details/masswildlife-calendar)

**JIRA ticket #:** DP-7482

**Comment by swunderlee** [comment]
MassDOT would also like an Agency-wide calendar of events and meetings. Is it possible to add functionality that would allow an individual posting an event to also add it to a 'master' calendar as well as to a page?

**Comment by RachaelELee** [comment]
DCR's Universal Access Program would also benefit from being able to create calendars, and having a single event appear on multiple calendars For example, a kayaking event might appear on a boating calendar with other boating events, as well as a full summer calendar with all events.

**Comment by estolarski** [comment]
We would love to have a calendar feature. I see that the original comment used suggestions from 2 of our pages (hunter education and MassWildlife calendar) Our angler education calendar is a good candidate as well.

**Comment by barb54** [comment]
DCR would also use the calendar for interpretive programs, public meetings and start/end dates for recreational such as camping, ice skating rinks, pools, ohv trails and more.

**Comment by joeg8612** [comment]
I'm curious what you may see as critical requirements for a calendar.  For example, if we display events in a grid that looks like a month, you can't fit very much into each day.  So can you explain what exactly you would like to see in a calendar feature?  Do you have some examples on other sites where you think this is done really well?

**Comment by barb54** [comment]
Here's an example Joe -  http://eeaonline.eea.state.ma.us/dcr/massparkspublic#/main/Calendar/

**Comment by estolarski** [comment]
Here is an example of a nice hunter education calendar:
https://register-ed.com/programs/colorado/98-colorado-hunter-education-course? We’re not looking for a full event registration solution like this example. But it would be nice to be able to clearly show (probably with color) whether a class is full, waitlist, or open.  It would also be nice to be able to display a short description and then expand below (instead of a whole new page) to show details. I like that Barb’s DCR example generates a link to share. I like the option for a map in the Colorado example. An option to download the event to Outlook or Google would be great too.

**Comment by Lchiong** [comment]
Yes please! The OIG would really love to have a calendar for our education program. Right now we use a Google calendar, but it has a lot of limitations. Visually it's not great. Currently not the best user experience, as you can see here: https://www.mass.gov/service-details/mcppo-calendar We could use a calendar with robust functionality that is a part of our CMS. Thanks!

**Comment by joeg8612** [comment]
Thanks for the examples, I'll note these in the backlog.
